docs/src/content/docs/ja/getting-started.mdx

```diff
@@ -0,0 +1,66 @@
+---
+title: 入門
+description: AstroのStarlightで新しいドキュメントサイトを作成する方法について学びます。
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+:::caution[作業中]
+Starlightはまだ開発の初期段階にあり、開発が進むにつれバグや変更が生じることが予想されます。何か不具合を見つけた場合は、[GitHubでissueを開く](https://github.com/withastro/starlight/issues/new/choose)か、または[Discord](https://astro.build/chat)でお知らせください。
```

Comment:
I shortened the first sentence a bit in 349dce1302c0f0f8eeb774127883eb6d701f60d9 to take up less space. Looks like it’s a clean edit in Japanese too?
```suggestion
Starlightはまだ開発の初期段階にあり。何か不具合を見つけた場合は、[GitHubでissueを開く](https://github.com/withastro/starlight/issues/new/choose)か、または[Discord](https://astro.build/chat)でお知らせください。
```